composeApp/src/commonMain/kotlin/com/github/familyvault/services/FileCabinetService.kt

```diff
@@ -152,7 +152,7 @@ class FileCabinetService(
         storeId: String?,
         limit: Long,
         skip: Long
-    ): List<ByteArray> {
+    ): List<Pair<ByteArray, Long>> {
```

Comment:
Dla bezpieczeństwa warto wydzielić to do innej metody a ten powinien zostać tak jak był w PrivMxClient, też zamiast Pair<...,...> powinien być jakiś model.